web/modules/custom/bnf/src/Plugin/bnf_mapper/BnfMapperNodePluginBase.php

```diff
@@ -96,20 +96,7 @@ public function getNode(NodeArticle|NodePage|NodeGoArticle|NodeGoCategory|NodeGo
    *   NOTICE: Unsupported paragraphs are skipped.
    */
   private function getParagraphs(NodeArticle|NodePage|NodeGoArticle|NodeGoCategory|NodeGoPage $object): array {
-    $paragraphs = [];
-    $objectParagraphs = $object->paragraphs ?? [];
-
-    foreach ($objectParagraphs as $paragraph) {
-      try {
-        $paragraphs[] = $this->manager->map($paragraph);
-      }
-      catch (\Exception $e) {
-        $this->logger->error('Unable to map paragraph of type - skipping: ' . $e->getMessage());
-      }
-    }
-
-    return $paragraphs;
-
+    return $this->manager->mapAll($object->paragraphs ?? []);
```

Comment:
What about the "skiping paragraphs that are not supported" logic, that you're removing?

From what i can see, you've not introduced that as part of mapAll() ?